include/exadg/utilities/exceptions.h

```diff
@@ -0,0 +1,48 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2022 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef INCLUDE_EXADG_UTILITIES_EXCEPTIONS_H_
+#define INCLUDE_EXADG_UTILITIES_EXCEPTIONS_H_
+
+#include <deal.II/base/exceptions.h>
+
+namespace ExaDG
+{
+/**
+ * The following exception is based on and motivated by dealii::ExcNotImplemented.
```

Comment:
Why do you write `DeclExceptionMsg` without `dealii::` namespace? This is exactly the kind of dependency that I criticized or questioned here https://github.com/exadg/exadg/pull/162#discussion_r806131679. I more and more come to the opinion that this should only be used internally in deal.II

Reply:
It's again a `#define`: https://github.com/dealii/dealii/blob/18a3861abce9e60d2a82e8ba52b8ed1a7c6774ed/include/deal.II/base/exceptions.h#L213-L230